### PR TITLE
Feature/view post

### PR DESCRIPTION
**Pull request by samra-pullflow**

### CODE REVIEW

File: api/src/lib/auth.ts [hunk 1]
```diff
@@ -7,6 +7,9 @@
  *
  * See https://redwoodjs.com/docs/authentication for more info.
  */
+
+//  api/src/lib/post_manager -> db calls to create/update/delete/get post
```
Comment (amnanwar) [inline api/src/lib/auth.ts]: dont need this comment here

File: api/src/lib/post_manager.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import type { QueryResolvers, MutationResolvers } from 'types/graphql'
+
+import { db } from 'src/lib/db'
```
Comment (amnanwar) [inline api/src/lib/post_manager.ts]: use relative imports

File: api/src/lib/post_manager.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+export const posts: QueryResolvers['posts'] = () => {
+  return db.post.findMany()
+}
+// postgresql://postgres:simplepw@localhost:5432/posts_db
```
Comment (amnanwar) [inline api/src/lib/post_manager.ts:7]: we can remove this

File: api/src/lib/posts.test.ts [hunk 4]
```diff
@@ -1,7 +1,13 @@
 import type { Post } from '@prisma/client'
-
-import { posts, post, createPost, updatePost, deletePost } from './posts'
-import type { StandardScenario } from './posts.scenarios'
+import {
```
Comment (amnanwar) [inline api/src/lib/posts.test.ts:2]: relative imports

Comment (amnanwar) [reply api/src/lib/posts.test.ts:2]: also this file should be called post_manager.test.ts

File: api/src/services/posts/posts.ts [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { db } from 'src/lib/db'
 export const posts: QueryResolvers['posts'] = () => {
   return db.post.findMany()
 }
-
+// postgresql://postgres:simplepw@localhost:5432/posts_db
```
Comment (amnanwar) [inline api/src/services/posts/posts.ts:8]: not needed

File: api/src/lib/post_manager.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import type { QueryResolvers, MutationResolvers } from 'types/graphql'
+
+import { db } from 'src/lib/db'
+
+export const posts: QueryResolvers['posts'] = () => {
```
Comment (amnanwar) [inline api/src/lib/post_manager.ts:4]: no need for query or mutation resolvers in this file.
you can create an object literal with methods defined inside like

export const Post: {
getPost: () => {},
getManyPosts: () => {},
etc
}

and use these in your post service

File: api/src/services/posts/posts.ts [hunk 7]
```diff
@@ -5,7 +5,7 @@ import { db } from 'src/lib/db'
...
 export const post: QueryResolvers['post'] = ({ id }) => {
   return db.post.findUnique({
```
Comment (amnanwar) [inline api/src/services/posts/posts.ts:10]: we shouldn't be doing any direct db calls in service, use methods from post_manager

File: web/src/components/ArticleCell/ArticleCell.tsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    article: post(id: $id) {
+      id
+      title
+      body
+      createdAt
+    }
+  }
+`
+
+export const Loading = () => <div>Loading...</div>
```
Comment (amnanwar) [inline web/src/components/ArticleCell/ArticleCell.tsx:20]: use chakra for loading, empty, and failure as well

File: web/src/pages/HomePage/HomePage.tsx [hunk 9]
```diff
@@ -1,17 +1,14 @@
-import { Heading } from '@chakra-ui/react'
-import { Text } from '@chakra-ui/react'
+// import { Heading } from '@chakra-ui/react'
```
Comment (amnanwar) [inline web/src/pages/HomePage/HomePage.tsx]: you can remove these if its not being used

**Review by amnanwar — CHANGES_REQUESTED** [review]

File: api/src/lib/post_manager.ts (hunk 6 above)
Comment (samra-pullflow) [reply api/src/lib/post_manager.ts:4]: handling in another branch

File: api/src/services/posts/posts.ts (hunk 7 above)
Comment (samra-pullflow) [reply api/src/services/posts/posts.ts:10]: handling in another branch

File: web/src/components/ArticleCell/ArticleCell.tsx (hunk 8 above)
Comment (samra-pullflow) [reply web/src/components/ArticleCell/ArticleCell.tsx:20]: handling in another branch

File: api/src/lib/posts.test.ts (hunk 4 above)
Comment (samra-pullflow) [reply api/src/lib/posts.test.ts:2]: handling in another branch

**Review by amnanwar — APPROVED** [review]
Looks good, please remove the two comments related to postgres in the next PR

**Pull request merged**